### ISSUE TITLE
Add "I don't care about cookies" list

### ISSUE THREAD

**Issue by KonoromiHimaries** [NanoAdblocker/NanoCore#99]
`https://www.i-dont-care-about-cookies.eu/abp/`
`https://www.i-dont-care-about-cookies.eu/`

https://github.com/gorhill/uBlock/issues/909

**Comment by kah0922** [comment]
From the official website:

> Be aware that the filterlist is not as effective as a browser extension but it will hide most of cookie warnings.

Since the extension is available on both Chrome and Firefox, I don't see the point of adding this to the default filter list.

**Comment by hawkeye116477** [comment]
But not everyone will want to install additional extension to work that Nano just can do rather with the same efficiency. This filterlist have many rules, much more than Fanboy's, so it's still useful.

**Comment by Emilylove00** [comment]
I don't care about cookies extension works fine here.

**Comment by jspenguin2017** [comment]
I would add this without question if we have a "filter store". The current system, 3rd party panel, is already long enough, I don't really want to add more filters unless it's reasonably useful. 

For this case, chances are the extension will work better as it has access to better script injection system.

**Comment by KonoromiHimaries** [comment]
> For this case, chances are the extension will work better as it has access to better script injection system.

i think you can update nano to compatibility with all of these rule : D

**Comment by jspenguin2017** [comment]
Is there a rule file or it's hard coded?

**Comment by jspenguin2017** [comment]
AFAIK this filter doesn't use extended syntax, how does it compare to Cookiemonster and Adguard's annoyance filter? Having those two, is this filter necessary?

**Comment by Ravizz** [comment]
Just some tech: adding `I don't care about cookies` (with `:not(body):not(html)` removed) gives 4800 (28%) more unique cookie filters:

```
Adguard’s Annoyance List      (8 544 from 10 242 used)  50,6%
Fanboy’s Cookiemonster List   (8 343 from  8 344 used)  49,4%
                                                 total 100,0%
I don't care about cookies    (4 800 from 11 237 used) +28,0%
```

**Comment by Atavic** [comment]
https://github.com/NanoAdblocker/NanoCore/issues/99#issuecomment-360683833

I'd go with Fanboy’s Annoyance, of which Fanboy's Cookiemonster is a subset. See: https://github.com/gorhill/uBlock/issues/2997#issuecomment-328782199

**Comment by Atavic** [comment]
Re: https://github.com/NanoAdblocker/NanoCore/issues/99#issuecomment-360683833

I'd go with Fanboy’s Annoyance, of which Fanboy's Cookiemonster is a subset.

See: https://github.com/gorhill/uBlock/issues/2997#issuecomment-328782199

**Comment by jspenguin2017** [comment]
Blocked by https://github.com/NanoAdblocker/NanoCore/issues/113

**Comment by Raaviz** [comment]
Also to add [Web Annoyances Ultralist](https://github.com/yourduskquibbles/webannoyances)

**Comment by jspenguin2017** [comment]
OK sure, but still blocked by #113 though.

**Comment by jspenguin2017** [comment]
Merged into https://github.com/NanoAdblocker/NanoCore/issues/113
I'll add `https://www.i-dont-care-about-cookies.eu/abp/` but after the new filter manager.

**Comment by Raavizz** [comment]
And [Web Annoyances Ultralist](https://github.com/yourduskquibbles/webannoyances) too.

**Comment by jspenguin2017** [comment]
Yea sure, why did you delete your comment though?
